hack/make-bundle-vars.mk

```diff
@@ -70,7 +70,7 @@ CSI_ADDONS_BUNDLE_IMG ?= $(CSI_ADDONS_IMAGE_REGISTRY)/$(CSI_ADDONS_REGISTRY_NAME
 CATALOG_IMG ?= $(IMAGE_REGISTRY)/$(REGISTRY_NAMESPACE)/$(CATALOG_IMAGE_NAME):$(IMAGE_TAG)
 
 # Produce CRDs that work back to Kubernetes 1.11 (no version conversion)
-CRD_OPTIONS ?= "crd:trivialVersions=true,preserveUnknownFields=false"
+CRD_OPTIONS ?= "crd:generateEmbeddedObjectMeta=true"
```

Comment:
could you pls mention what these change in options mean?

Reply:
the controllern-gen version 0.9.2 doesn't have the crd options: `crd:trivialVersions=true,preserveUnknownFields=false`.
From the [doc](https://book.kubebuilder.io/reference/controller-gen#generators):
```
schemapatch
generateEmbeddedObject - patches existing CRDs with new schemata.
It will generate output for each "CRD Version" (API version of the CRD type itself) , e.g. apiextensions/v1) available.
```